quill-core/src/main/scala/io/getquill/util/Messages.scala

```diff
@@ -4,6 +4,11 @@ import scala.reflect.macros.blackbox.{ Context => MacroContext }
 
 object Messages {
 
+  private val debugEnabled =
```

Comment:
I actually updated the PR and added a ENV variable. Also I flipped the switch, so that logging is now enabled by default, but could be enabled by either setting env or a system property.
Well the best way would be having a way to set the Log Level, but at the moment I don't know any possibility to do so with compile time macro's at least you could disable them with `logLevel := Level.Warn` but unfortunatly that disable's also all info logs of the scala compiler, so it would be really really great to have a log level for the quill code only, but at the moment I don't know any "good" way to do it.

-- 

Maybe that's the better solution. But well I actually think it's definitly not the most important feature.